### PR TITLE
Prevent spam with /op

### LINKED ISSUE #887: Disallow opping already opped players
To avoid the spam caused by trollers spamming /op, we should made a restriction to the /op command and not allow people OP players that are already opped.

I have this feature on my server: 
https://github.com/ImmaFreedom-Dev/ImmaFreedomMod/blob/master/src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java#L59-#L66

### PR DESCRIPTION
**Pull request by OutOfYourMind**
Since #887 had this, I figured to make a pull request. I give full credit to ImmaFreedom's Developers.

EDIT: Original issue: https://github.com/TotalFreedom/TotalFreedomMod/issues/538

### CODE REVIEW

**Comment by JeromSar** [comment]
@OutOfYourMind The PR is good, but the issue was closed. The original issue (https://github.com/TotalFreedom/TotalFreedomMod/issues/538) requires that you also need to block admins from using /opall when there are no players to be opped. Could you implement that in this PR to confirm to the original issue?

**Comment by OutOfYourMind** [comment]
Yes, will do.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java [hunk 1]
```diff
@@ -51,8 +51,15 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
...
-        TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
-        player.setOp(true);
+        if (!player.isOp())
+        {
+            TFM_Util.adminAction(sender.getName(), "Opping " + player.getName(), false);
+            player.setOp(true);
+        }
+        else
+        {
+            playerMsg("That player is already OP.");
```
Comment (WickedGamingUK) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_op.java]: This should read "That player is already an OP." - we might as well use correct English ;)

**Comment by OutOfYourMind** [comment]
See #890

**Comment by OxLemonxO** [comment]
Please use correct formatting.

**Comment by OxLemonxO** [comment]
Plus even if all players are op, they should still be gamemode changed if it's -c or -s

**Comment by LegendIsAwesomes** [comment]
@OutOfYourMind I don't appreciate you using the commits I made and using them on TFM claiming them as your own.

**Comment by OutOfYourMind** [comment]
@LegendIsAwesomes I gave credit to full developers, once again.

**Comment by Wild1145** [comment]
The bottom line here is that any code you commit should be your own work. Simple as that.

**Comment by JeromSar** [comment]
This PR will not be merged. The contribution is question is not your own and you do not have the jurisdictional capabilities to license said code under [TFGL](https://github.com/TotalFreedom/License).

@LegendIsAwesomes

**Comment by LegendIsAwesomes** [comment]
Why was I tagged @JeromSar

**Comment by JeromSar** [comment]
@LegendIsAwesomes You originally mentioned that the author used your commits without your express permission (see comments above).

**Comment by LegendIsAwesomes** [comment]
Oh okay, I am guessing your asking for my permission or what (idk what a few words above mean)

**Comment by JeromSar** [comment]
@LegendIsAwesomes No, I have closed this PR. I only accept PRs if they're made by the original author.

**Pull request closed without merging**